kura/org.eclipse.kura.wire.provider/src/main/java/org/eclipse/kura/internal/wire/WireServiceImpl.java

```diff
@@ -265,7 +268,7 @@ public void deleteWireConfiguration(final WireConfiguration wireConfiguration) {
                 }
             }
         } catch (final InvalidSyntaxException e) {
-            throw new KuraRuntimeException(KuraErrorCode.INTERNAL_ERROR, ThrowableUtil.stackTraceAsString(e));
+            throw new IllegalArgumentException(ThrowableUtil.stackTraceAsString(e));
```

Comment:
The client code cannot do anything about `InvalidSyntaxException` (a checked one). Especially if a checked exception is used for conditions from which the caller cannot recover, I think it is justified to translate that exception to a runtime exception.